CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+# Contributing
+
+You will find here the conventions and rules to publish new blog posts and to develop new blog features and enhancements.
+
+Please,
+- [open an issue](https://github.com/src-d/blog/issues) to request any new feature or to report a bug.
+- [create a new PR](https://github.com/src-d/blog/pulls) to request the review of a new post or feature.
+
+
+## Table of contents
+
+<!-- TOC -->
+
+- [Contributing](#contributing)
+    - [Table of contents](#table-of-contents)
+    - [Preview the blog posts](#preview-the-blog-posts)
+    - [Creating a new post](#creating-a-new-post)
+        - [Content schema](#content-schema)
+        - [Front matter](#front-matter)
+        - [Formatting the content](#formatting-the-content)
+        - [Links to other source{d} blog posts](#links-to-other-sourced-blog-posts)
+        - [Assets](#assets)
+        - [Authors](#authors)
+    - [Peer review](#peer-review)
+    - [How to publish blog posts and deploy the blog](#how-to-publish-blog-posts-and-deploy-the-blog)
+
+<!-- /TOC -->
+
+
+## Preview the blog posts
+
+While you are developing or creating new contents for the blog, and always before publishing a PR, you should validate your changes locally, specially if you're modifying common features or you're using shortcodes.
+
+To locally serve the blog, you need to satisfy the [project requirements](README.md#requirements), and then to run:
+
+```shell
+make serve;
+```
+and then, go to [http://localhost:8484](http://localhost:8484)
+
+
+## Creating a new post
+
+Posts are stored under [`post` content directory](content/post)
+
+A blog post content stored under: `content/post/__POST_FILE_NAME__.md`
```

Comment:
These two lines might make user think that `content` and `post` are something different.

How about changing it to something like

```
To create a new blog post, create new a file under `content/post/__POST_FILE_NAME__.md` using the following content schema.
```